python/ray/_private/ray_constants.py

```diff
@@ -264,7 +264,17 @@ def to_memory_units(memory_bytes, round_up):
     f"ray::DELETE_{WORKER_PROCESS_TYPE_RESTORE_WORKER_NAME}"
 )
 
-LOG_MONITOR_MAX_OPEN_FILES = 200
+# The number of files the log monitor will open. If more files exist, they will
+# be ignored.
+LOG_MONITOR_MAX_OPEN_FILES = int(
+    os.environ.get("RAY_LOG_MONITOR_MAX_OPEN_FILES", "200")
```

Comment:
Can we change the defaults here to max_open=1, num_lines=1000?

Reply:
we can increase num_lines, but `max_open=1` will make us ignore all but 1 worker file.